### ISSUE TITLE
Music Mixer stops when doing fullscreen toggle

### ISSUE THREAD

**Issue by hyarsan** [libretro/RetroArch#6513]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

When you press F ( or whatever you set the Fullscreen Toggle to ) while music is playing in the music mixer, the music that is playing stops.

### Expected behavior

I expect it to just continue playing the music

### Actual behavior

The music stops when fullscreen toggling

### Steps to reproduce the bug

1. Load game ( doesn't matter what content or what core but I used Snes9x to make sure )
2. Go to music tab and "Add to mixer"
3. Fullscreen toggle and you realize the music stops

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.2 (latest nightly)

### Environment information

- OS: Windows 10 64-bit
- Compiler: MinGW 64-bit from Buildbot

**Comment by orbea** [comment]
I'd guess this is a consequence of resetting the context. Easier said than done, but not destroying the entire window when toggling full screen would be an improvement. I'd suggest looking at mpv for ideal behavior in this case.

**Comment by twinaphex** [comment]
Yeah, that is currently one of the limitations of the audio mixer.
